Livecheckables/opencv@2.rb

```diff
@@ -1,4 +1,4 @@
 class OpencvAT2
-  livecheck :url   => "https://github.com/opencv/opencv/releases",
-            :regex => %r{href="/opencv/opencv/tree/(2[0-9\.]+)"}
+  livecheck :url   => "https://github.com/opencv/opencv.git",
+            :regex => /^2.4.(\d+(?:\.\d+)+)$/
```

Comment:
```suggestion
            :regex => /^(2(?:\.\d+)+)$/
```

We only want to restrict the major version here (to `2`). Though 2.4.x is the latest version, we don't want to rule out the chance of there being 2.5+ versions.

In this case, you simply substitute `2` for the first `\d+` part of the regex, as that's where the major version is.

Reply:
good call